src/main/drivers/display_font_metadata.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include <stdbool.h>
+#include <stdint.h>
+
+typedef struct osdCharacter_s osdCharacter_t;
+
+typedef struct displayFontMetadata_s {
+    uint8_t version;
+    uint16_t charCount;
+} displayFontMetadata_t;
+
+// 'I', 'N', 'A', 'V'
```

Comment:
@saidinesh5 , what is the purpose of this section? with "INAV"

Reply:
I had to include this for availing `displayFontMetadata_t` this one function (which is used nowhere). Originally I wanted to keep this osd driver as close to the inav version as possible to minimize the headaches when updating, but I am beginning to think that we don't have to worry about that right now because we have a lot of code that is useless currently.

```
static bool getFontMetadata(displayFontMetadata_t *metadata, const displayPort_t *displayPort)
{
    UNUSED(displayPort);
    metadata->charCount = 512;
    metadata->version = FONT_VERSION;
    return true;
}
```

Reply:
i believe, this will need to be EMUF for hdzero font `EMUF_000.bmp`. not validated yet.